lib/modules/platform/gerrit/readme.md

```diff
@@ -64,7 +64,7 @@ For example, if you want to use the [Merge Confidence](https://docs.renovatebot.
 Sometimes the PR title passed to the Gerrit platform code is different from the first line of the commit message.
 For example:
 
-Commit-Message=`Update keycloak.version to v21` \
-Pull-Request-Title=`Update keycloak.version to v21 (major)`
+- Commit-Message=`Update keycloak.version to v21`
+- Pull-Request-Title=`Update keycloak.version to v21 (major)`
```

Comment:
Hi @NiasSt90, you worked on the Gerrit platform support. Our published docs don't display the current commit message/pull request title thing properly, or at least I find it hard to read:

![renovate-gerrit-platform-docs-rendering](https://github.com/renovatebot/renovate/assets/34918129/e42ad39a-22e7-42d9-bb6f-b2bba9e586fb)

I think a list is better than spreading the example over a single line. Can you review my changes, and tell me what you think?